src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmPolicyAliases.md

```diff
@@ -0,0 +1,218 @@
+---
+external help file: Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml
+Module Name: AzureRM.Resources
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmPolicyAliases
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@mentat9 please make sure there is content in the template sections provided (synopsis, description, examples)